api/clients/eigenda_client.go

```diff
@@ -291,7 +348,7 @@ func (m *EigenDAClient) putBlob(ctx context.Context, rawData []byte, resultChan
 				return
 			default:
 				// this should never happen. If it does, the blob is in a heisenberg state... it could either eventually get confirmed or fail
-				errChan <- fmt.Errorf("unknown reply status %d. ask for assistance from EigenDA team, using requestID %s", statusRes.Status, base64RequestID)
+				errChan <- api.NewErrorInternal(fmt.Sprintf("unknown reply status %d. ask for assistance from EigenDA team, using requestID %s", statusRes.Status, base64RequestID))
```

Comment:
Same here, if EigenDA logics is so wrong at this point, should we just let the client to use EthDA?

Reply:
This one I'm not sure... actually do you know how we can be in this state? Not sure that the heisenberg state means. It sounds like its not a problem with eigenda but possibly just some weird logic with this particular blob, so just resubmitting the blob should be the correct behavior?

Reply:
yeah, internalError should be good. There is something wrong, but nothing shows that it is catastrophic